### DISCUSSION TITLE
How to display map

### DISCUSSION THREAD

**Discussion by wormvortex** [flobz/psa_car_controller#646] [General]
I've set it up via docker and all seems to be working. However I'm confused how to display a map. It says:

```
You can add the -r argument to record the position of the vehicle and retrieve this information in a dashboard.

python3 psa-car-controller -f config.json -c charge_config.json -r

You will be able to visualize your trips, your consumption and some statistics:
```

I do not understand what this means or what I need to do. Can someone help :D

**Comment by sod75** [comment]
There's an issue open for it
https://github.com/flobz/psa_car_controller/issues/637

**Reply by wormvortex** [reply]
But would I still not have to set it up orginallly?

My issue was I do not understand what it is asking with adding the -r argument to record the position....

**Comment by sod75** [comment]
As far as I can tell it's part of the default options in the docker image already

 PSACC_OPTIONS  -c -r --web-conf

which can be altered by the environment variable if needed, ( but I don't think it is)
